packages/manager-react-components/package.json

```diff
@@ -77,7 +77,7 @@
     "@types/react-dom": "18.2.7",
     "@typescript-eslint/eslint-plugin": "^4.7.4",
     "@vitejs/plugin-react": "4.0.4",
-    "@vitest/coverage-v8": "0.34.1",
+    "@vitest/coverage-v8": "^2.1.9",
```

Comment:
The bump is being handled as part of a dedicated ticket - https://github.com/ovh/manager/pull/15922/files#diff-67b8a223598086ec961e7875d48f79f30a64cd6c42551a033132f51fd6ed55aa
Can you remove this change?

Reply:
it's fixed but the yarn.lock have still change even if no change in package.json